chapter-1/exercises/1.10.js

```diff
@@ -0,0 +1,126 @@
+// Exercise 1.10. The following function computes a mathematical function called Ackermann’s function.
+function A(x,y) {
+   if (y === 0) 
+        return 0;
+   else if (x === 0) 
+        return 2 * y;
+   else if (y === 1)
+        return 2;
+   else return A(x - 1, A(x, y - 1));

```

Comment:
Please write it on two lines, with proper indentation. Like:
```
else 
    return A(x - 1, A(x, y - 1));
```
I know that the book wrote it this way. But you should be more strict ;)